### ISSUE TITLE
Training set

### ISSUE THREAD

**Issue by hoanganh-2704** [Xiyue-Wang/1st-in-MICCAI-MIDOG-2021-challenge#5]
Hi, I have a question about MIDOG2021 training set. Do you apply Hovernet to generate mask for the fourth scanner (Leica GT450) (which are not provided with mitosis annotations) or this scanner was only used in  Fourier-based data augmentation step. Moreove, how did you deal with hard negative samples in the dataset (provided groundtruth bbox but not mitosis)? Thank you.

**Comment by Xiyue-Wang** [comment]
Hi,1 . fourth scanner  was only used in Fourier-based data augmentation step .But you can also generate pseudo-label
2. Regarding the hard negative samples, we did not deal with them, we think that with the fine mask, the model should be able to learn the distinguishing features very well

**Comment by hoanganh-2704** [comment]
Can I ask what is the value of gamma did you use for FocalLoss() function?
